lib/sample_uploader/utils/importer.py

```diff
@@ -24,6 +25,42 @@
 NOOP_VALS = ['ND', 'nd', 'NA', 'na', 'None', 'n/a', 'N/A', 'Na', 'N/a', '-']
 
 
+def find_header_row(sample_file, file_format):
```

Comment:
could you add some brief documentation about what this function looks for when its trying to find the header row